app/controllers/facility_reservations_controller.rb

```diff
@@ -121,8 +121,10 @@ def new
 
   # POST /facilities/:facility_id/instruments/:instrument_id/reservations
   def create
-    @instrument   = current_facility.instruments.find_by_url_name!(params[:instrument_id])
-    @reservation  = @instrument.reservations.new(params[:reservation])
+    @instrument =
+      current_facility.instruments.find_by!(url_name: params[:instrument_id])
+    @reservation =
+      @instrument.admin_reservations.new(params[:reservation].merge(split_times: true))
```

Comment:
Can we use the `assign_times_from_params`? I think that's what we've been using in most places in the application. The `split_times: true` is mostly in the specs.